### ISSUE TITLE
Camera pose for triplane

### ISSUE THREAD

**Issue by 87003697** [MrTornado24/Next3D#3]
Hi， I appreciate your excellent work and I have some questions about the triplane configuration. In my own work, I found the pre-extracted camera pose of EG3D ( the original triplane work) appears increasingly inaccurate when the head rotation increases. So my question is, did you follow the original scripts in EG3D to extract the camera poses, or do you make other attempts?

**Comment by MrTornado24** [comment]
Yes, we follow the original scripts to extract head poses. BTW, it would be appreciated if you can cancel email-auto-reply for every issue. Thanks.

**Comment by Gong-Yicheng** [comment]
I have encountered a similar issue. I tried preprocessing the Obama image array using the original code, but it did not work as expected.

**Comment by sunshineatnoon** [comment]
Hi, @MrTornado24 do you need any special alignment between the FLAME mesh and the EG3D camera? It seems that they use different coordinate system.

**Comment by Dancingmader** [comment]
> I have encountered a similar issue. I tried preprocessing the Obama image array using the original code, but it did not work as expected.

1. Comment line 120-126 in dataset_preprocessing/ffhq/preprocess_face_cameras.py, cause we don't need that in wild image preprocessing. The authors use these codes maybe for dataset augmentation. ;-)
2. Comment reenact_avatar_next3d.py line 86:
`@click.option('--num_frames', help='number of testing frames', type=int, required=False, metavar='BOOL', default=500, show_default=True) `, 
and set
`num_frames = len(img_list)`
in line 125;
change the first if condition to `if k >= num_frames-1: break`
in line 127 to prevent list index out of range error.
